### PR TITLE
Google oauth sign-in

### PR DESCRIPTION
**Pull request by andkamau**
Replaced sign-in on web interface from [investigativedashboard.org](https://investigativedashboard.org/) to [google](google.com)

### CODE REVIEW

File: contrib/docker_settings.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 ELASTICSEARCH_URL = os.environ.get('ALEPH_ELASTICSEARCH_URI')
 
 SQLALCHEMY_DATABASE_URI = os.environ.get('ALEPH_DATABASE_URI')
-PREFERRED_URL_SCHEME = 'https'
+PREFERRED_URL_SCHEME = 'http'
```
Comment (DavidLemayian) [inline contrib/docker_settings.py]: Preference should always be `https`

File: docker-compose.yml [hunk 2]
```diff
@@ -63,7 +63,7 @@ web:
       - "/srv/data/live/aleph-logs/worker:/var/log"
     environment:
       ALEPH_ELASTICSEARCH_URI: http://aleph_elasticsearch:9200/
-      ALEPH_DATABASE_URI: postgresql://aleph:aleph@aleph_postgres/aleph
+      ALEPH_DATABASE_URI: postgresql://aleph:aleph@localhost/aleph
```
Comment (DavidLemayian) [inline docker-compose.yml]: `aleph_postgres` can be pointed to 127.0.0.1 or anywhere in the `/etc/hosts` files so let's revert this change?

File: aleph/default_settings.py [hunk 3]
```diff
@@ -46,8 +46,8 @@
 CELERY_IMPORTS = ('aleph.queue')
 
 OAUTH = {
-    'consumer_key': '',
-    'consumer_secret': '',
+    'consumer_key': 'bcee20a7223a6ae26aea49a392bd21d3c4cbe075',
+    'consumer_secret': 'wTEEOYW5b66EumjZ_XHNh4Yv',
```
Comment (DavidLemayian) [inline aleph/default_settings.py]: Any reason why you'd share these?

File: aleph/templates/layout.html [hunk 4]
```diff
@@ -37,9 +42,29 @@
...
+                    console.log("Image URL: " + profile.getImageUrl());
+                    console.log("Email: " + profile.getEmail());
+
+                    // The ID token you need to pass to your backend:
+                    var id_token = googleUser.getAuthResponse().id_token;
+                    console.log("ID Token: " + id_token);
+                  };
+                </script> 
+              -->
+
```
Comment (DavidLemayian) [inline aleph/templates/layout.html]: Any reason to leave this on here?

**Comment by DavidLemayian** [comment]
Great first move. See the comments above and revert.

File: contrib/docker_settings.py (hunk 1 above)
Comment (andkamau) [inline contrib/docker_settings.py]: :+1:

File: docker-compose.yml (hunk 2 above)
Comment (andkamau) [inline docker-compose.yml]: :+1:

File: aleph/default_settings.py (hunk 3 above)
Comment (andkamau) [inline aleph/default_settings.py]: Nope. Removed.

File: aleph/templates/layout.html (hunk 4 above)
Comment (andkamau) [inline aleph/templates/layout.html]: cleaned this up
